src/Controllers/CurrentUsersController.cs

```diff
@@ -19,7 +19,7 @@ public class CurrentusersController : BaseController<User>
             UserService userService)
          : base(loggerFactory, jsonApiContext, resourceService, currentUserContext, organizationService, userService)
         { }
-
+        #pragma warning disable 1998
```

Comment:
not sure what pragma 1998 is all about? Should there be a comment here?

Reply:
It's to make it stop complaining about the async function not being async...the function is an override that throws a "not implemented" error because you can't do anything with currentuser.  I will add a comment.